openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -0,0 +1,376 @@
+
+        # Update 'data' different in zou DB
+        updated_data = {
+            k: v for k, v in item_data.items() if item_doc["data"].get(k) != v
+        }
+        if updated_data or not item_doc.get("parent"):
+            assets_with_update.append(
+                (
+                    item_doc["_id"],
+                    {
+                        "$set": {
+                            "name": item["name"],
+                            "data": item_data,
+                            "parent": asset_doc_ids[item["project_id"]]["_id"],
+                        }
+                    },
+                )
+            )
+
+    return assets_with_update
+
+
+def write_project_to_op(project: dict, dbcon: AvalonMongoDB) -> UpdateOne:
+    """Write gazu project to OP database.
+    Create project if doesn't exist.
+
+    Args:
+        project (dict): Gazu project
+        dbcon (AvalonMongoDB): DB to create project in
+
+    Returns:
+        UpdateOne: Update instance for the project
+    """
+    project_name = project["name"]
+    project_doc = dbcon.database[project_name].find_one({"type": "project"})
+    if not project_doc:
+        print(f"Creating project '{project_name}'")
+        project_doc = create_project(project_name, project_name, dbcon=dbcon)
+
+    # Project data and tasks
+    project_data = project["data"] or {}
+
+    # Build project code and update Kitsu
+    project_code = project.get("code")
+    if not project_code:
+        project_code = project["name"].replace(" ", "_").lower()
+        project["code"] = project_code
+
+        # Update Zou
+        gazu.project.update_project(project)
+
+    # Update data
+    project_data.update(
+        {
+            "code": project_code,
+            "fps": project["fps"],
+            "resolutionWidth": project["resolution"].split("x")[0],
+            "resolutionHeight": project["resolution"].split("x")[1],
+        }
+    )
+
+    return UpdateOne(
+        {"_id": project_doc["_id"]},
+        {
+            "$set": {
+                "config.tasks": {
+                    t["name"]: {"short_name": t.get("short_name", t["name"])}
+                    for t in gazu.task.all_task_types_for_project(project)
+                },
+                "data": project_data,
+            }
+        },
+    )
+
+
+def sync_all_project(login: str, password: str):
+    """Update all OP projects in DB with Zou data.
+
+    Args:
+        login (str): Kitsu user login
+        password (str): Kitsu user password
+
+    Raises:
+        gazu.exception.AuthFailedException: Wrong user login and/or password
+    """
+
+    # Authenticate
+    if not validate_credentials(login, password):
+        raise gazu.exception.AuthFailedException(
+            f"Kitsu authentication failed for login: '{login}'..."
+        )
+
+    # Iterate projects
+    dbcon = AvalonMongoDB()
+    dbcon.install()
+    all_projects = gazu.project.all_projects()
```

Comment:
It could decrease the data to be synchronized
```suggestion
    all_projects = gazu.project.all_open_projects()
```
And maybe switch closed projects to inactive project in OP.

Reply:
> And maybe switch closed projects to inactive project in OP.

This would be useful.